src/devices/vdml_imu.c

```diff
@@ -22,7 +23,11 @@
 		return_port(port - 1, err_return);                                         \
 	}
 
-int32_t imu_reset(uint8_t port) {
+typedef struct imu_reset_data { 
```

Comment:
Have you double checked that this fits in the registry? You may also want to pack the struct.

Reply:
Struct packed, I'm pretty sure it fits in the registry (6 doubles at 8 bytes each being 48 bytes total, fitting into 128 bytes on the pad in the registry).